dockerfile-suse.template

```diff
@@ -0,0 +1,40 @@
+# Copyright 2022 IBM Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM __BASE_IMAGE__
+ARG ARCH
+ARG DISTRO_NAME
+ARG DISTRO_NICK
+ARG DISTRO_RELEASE
+ARG AT_VERSION
+ARG EXTRA
+ARG MINOR
+ARG REPO
+
+WORKDIR /root
+
+RUN zypper refresh \
+    && zypper install -y __BASE_PACKAGES__
+
+RUN rpm --import ${REPO}/${DISTRO_NAME}/SLES_${DISTRO_NICK}/gpg-pubkey-6976a827-5164221b \
+    && zypper addrepo ${REPO}/suse/SLES_15/ AT \
+    && zypper install -y __AT_PACKAGES__ \
+    && zypper clean
+
+RUN ln -sf /opt/at${AT_VERSION}${MINOR}${EXTRA}/lib64/ld64.so.2 /lib64/ld64.so.2
+RUN ln -sf /opt/at${AT_VERSION}${MINOR}${EXTRA}/sbin/ldconfig /sbin/ldconfig
+
+ENV PATH /opt/at${AT_VERSION}${MINOR}${EXTRA}/bin:$PATH
```

Comment:
Just a thought, for your consideration...

We have environment module files included with AT, although they may not be well known.  They can be used to set up an environment to run AT, and today they set not only PATH, as you have done here, but also MANPATH and INFOPATH, which are convenient.  If new settings arise in the future, we could add those, too.  The disadvantage is that another package must be installed before invoking the command shell:  environment-modules.

If the environment-modules package is installed, then appending `module load at${AT_VERSION}${MINOR}${EXTRA}` to `.bash_profile` I think would be sufficient.

Reply:
@ThinkOpenly Good point. Fixed. I also fixed some symlinks that were not pointing to the correct files.